tests/arch/x86/pagetables/testcase.yaml

```diff
@@ -3,3 +3,5 @@ tests:
     arch_allow: x86
     tags: userspace mmu
     filter: CONFIG_MMU
+    extra_configs:
+      - CONFIG_COMMON_LIBC_MALLOC_ARENA_SIZE=0
```

Comment:
This may not be needed if you can set the malloc arena size to 0 for PICOLIBC.

Reply:
see above.